tests/suites/cassandra_sanity/sanity_test.go

```diff
@@ -49,7 +49,12 @@ var _ = Describe(TestName, func() {
 	It("Installs the latest operator from the package registry", func() {
 		// TODO(mpereira) Assert that it isn't running.
 		err := kudo.InstallOperator(
-			OperatorName, TestNamespace, TestInstance, []string{},
+			OperatorName, TestNamespace, TestInstance, []string{
+				"NODE_CPU_MC=200",
+				"NODE_MEM_MIB=800",
+				"PROMETHEUS_EXPORTER_CPU_MC=100",
+				"PROMETHEUS_EXPORTER_MEM_MIB=200",
```

Comment:
These are probably development settings for local testing?

Reply:
right! I will remove these :ballot_box_with_check: from here and tls test

Reply:
but ts a good practice to have tests that consume as less resource as possible, I added these as the default ones where too large to run in konovy cluster with default config.
It can help in starting up running tests in small clusters faster, plus if we have extra pods in tests like KDC or like a client pod for testing out application features then those can fail to run due to resource starvation on the k8s nodes WDYT @ANeumann82 ?

Reply:
Yeah, I'm fine with that. I didn't know Cassandra enough yet to tune the settings down, but as long as we don't make anything flaky because Cassandra doesn't have enough CPU or Mem, i'm fine with having lower values for the tests.

Reply:
These settings are too low, and can likely cause resource throttling issues. I've seen it happen during MWTs. Unless we're specifically testing CPU/MEM parameters, let's use the default parameter values.